Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/BrushToolView.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ *  Paintroid: An image manipulation application for Android.
+ *  Copyright (C) 2010-2021 The Catrobat Team
+ *  (<http://developer.catrobat.org/credits>)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.ui.tools
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import android.graphics.BitmapShader
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Shader
+import android.util.AttributeSet
+import android.view.View
+import org.catrobat.paintroid.R
+import org.catrobat.paintroid.tools.ToolType
+import org.catrobat.paintroid.tools.options.BrushToolOptionsView.OnBrushPreviewListener
+import org.catrobat.paintroid.tools.options.BrushToolPreview
+
+private const val CONSTANT_1 = 3f
+private const val CONSTANT_2 = 16f
+private const val CONSTANT_3 = 56f
+private const val CONSTANT_4 = .25
```

Comment:
Since these names are not really helpful, please add the value to the name.
CONSTANT_3F or something like that.

Reply:
done :)